lib/install.ps1

```diff
@@ -679,14 +679,16 @@ function create_shims($manifest, $dir, $global, $arch) {
         $target, $name, $arg = shim_def $_
         write-output "Creating shim for '$name'."
 
-        # check valid bin
-        $bin = "$dir\$target"
-        if(!(is_in_dir $dir $bin)) {
-            abort "Error in manifest: bin '$target' is outside the app directory."
+        if(test-path "$dir\$target") {
+            $bin = "$dir\$target"
+        } elseif(test-path $target) {
+            $bin = $target
+        } else {
+            $bin = search_in_path $target
         }
         if(!(test-path $bin)) { abort "Can't shim '$target': File doesn't exist."}
```

Comment:
This needs to be something like `if(!$bin) { ... }` instead. Currently, if we don't find the file, `$bin` is null and this check blows up, causing lots of errors as the null value propogates. 
Also there's no need to do the `test-path` again.

Reply:
Yeah, you're right.